src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1633,6 +1646,11 @@ public override BoundNode VisitAssignmentOperator(BoundAssignmentOperator node)
             VisitLvalue(node.Left);
             VisitRvalue(node.Right);
 
+            if (node.RefKind != RefKind.None)
```

Comment:
I am not sure, but I cannot rule out completely that this is reachable. It will certainly be reachable if we allow ref re-assignments or if we run this analysis on lowered code.